src/components/templates/Camp/components/MapRoomOverlay.tsx

```diff
@@ -196,14 +193,14 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
 
             <div className="camp-venue-actions">
               {/* TODO: extract this into a generalised link component that handles internal/external correctly? */}
-              <a
+              <button
                 className="btn btn-block btn-small btn-primary"
-                onClick={onClick3}
-                href={currentRoomUrl}
-                {...getExtraLinkProps(isRoomExternal)}
+                onClick={onJoinRoom}
+                // href={currentRoomUrl}
+                // {...getExtraLinkProps(isRoomExternal)}
```

Comment:
Looks like invalid JSX to me.

Reply:
It's valid to comment code within a component tag in JSX like this.